### PR TITLE
Ions implemented for FOX

### PR DESCRIPTION
**Pull request by tpersson**
Ions implemented in the FOX part 
Fixes the proton mass in the wire that was not the same as in the rest of the code #443 
Initialize all the ions to the default ion chosen in the ion block. 

Numerical differences in many canoncial. They have NOT been updated. It would be good if someone else could have a look and see if this is just due to the fact that now the variable (mtc/(1+delta)) is the variable that is mainly used compared to delta before.

### CODE REVIEW

**Comment by JamesMolson** [comment]
i.e. oidpsv goes to moidpsv

Also could you merge in dev again - the merge conflict is just a "use mod_hions" so easy to fix.

File: SixTrack/sixtrack.s90 [hunk 1]
```diff
@@ -10750,8 +10764,8 @@ subroutine daten
       iskew=0
       preda=c1m38
 !hion
-      zz0 = 1
-      aa0 = 1
+      zz0 = one
```
Comment (JamesMolson) [inline SixTrack/sixtrack.s90]: these are integers, so using "1" is totally ok. "one" is a floating point variable and there will be a cast taking place with this.

**Comment by rdemaria** [comment]
I think a way to test the DA part is to look at the tunes written in fort.6
after the closed orbit search. Normally fort.6 is not checked by the test
suits, but probably the tune in fort.10 are the one from this routine. Can
you check that tunes did not change?

**Comment by tpersson** [comment]
Yeah that is perfectly fine. This how it was done for the tracking part. So to be consisten between the rwo implementations, or at least more consistent, I continued the same way.

Cheers
Tobias






-------- Original message --------

**Comment by tpersson** [comment]
I will just have to fix the ion tests and then this can be merged.

**Comment by tpersson** [comment]
All the tests should now pass and this fixes both the implementation of the ions and the proton mass in the wire #443. 

Should be possible to merge this now.

**Pull request merged**